CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # Change Log
 
+## [1.0.1] - 2020-01-10
+
+### Fixed
+
```

Comment:
I don't know how complete this list must be. I' missing #1797 and #1764.

Reply:
I missed them somehow. Thanks for pointing it out! It's now fixed.